docs/config-file-ort-yml.md

```diff
@@ -128,8 +128,8 @@ scopes defined in the examples below match the scopes in your project.
 ### When to Use Curations
 
 License finding curations should be used when you want to correct the licenses detected in the source code of the
-project. To define path excludes on a global level for third-party packages, please use the
-[package configurations](config-file-package-configuration-yml.md).
+project. To define curations on global level for third-party packages, please use 
```

Comment:
IIUC the term "repository package-curations " is new. 
I don't see need for introducing new terminology here, so I propose to not do that.

Reply:
Why does it talk about "package-**curations**"?